frontend/src/components/grantaccess/GrantApplicationAdmin.vue

```diff
@@ -75,10 +75,22 @@ const handleSubmit = async () => {
         })
         .catch((error) => {
             if (error.response?.status === 409) {
-                setNotificationMsg(
-                    Severity.warning,
-                    error.response.data.detail
-                );
+                // This is not a good way to check and replace a message error.
+                // Maybe we could implement a detail code in the backend's response,
+                // similar to what we have for 'self_grant_prohibited'
```

Comment:
Hi Nick, this is a good idea, I added it to the ticket #820, thanks!!

Reply:
Didn't know that message is from backend?!
I think long time ago we created a ticket to discuss situation like this that how we really like to unify error handling backend and frontend; but we never did have chance on that discussion.

Please add a //TODO with nosonar on your comment so we don't ignore this comment. Thanks.

Reply:
I pushed a fix, think for now 409 is just used for conflict, so did some cleanup, we can keep improve later